src/components/RequestRepository/index.tsx

```diff
@@ -52,60 +52,92 @@ const RequestRepository = () => {
           );
         }
       },
+      ...(activeTable === 'closed'
```

Comment:
This is a little JS trick to conditionally add items to an array.

Depending on the result of the ternary condition, the array will be spread/flattened onto the `columns` array.

Reply:
This logic is really hard to read at a glance. It's kind of hard to read after three glances. Do we need to split these tables into two components that are easier to read and maybe find some other way to share code between them?